src/components/TrapFocus/tests/TrapFocus.test.tsx

```diff
@@ -112,6 +112,7 @@ describe('<TrapFocus />', () => {
   it('keeps focus on nodes contained inside trap focus during mount', () => {
     const trapFocus = mountWithApp(
       <TrapFocus>
+        <button>First node</button>
```

Comment:
The test wasn't working correctly because the first element inside `TrapFocus` was being focused. Which was the input, regardless of `autoFocus`